examples/deployment/src/asa-helpers.js

```diff
@@ -0,0 +1,163 @@
+// Authors of example: Invertisment & amityadav0
+const algosdk = require('algosdk');
+
+// Function used to print created asset for account and assetid
+exports.printCreatedAsset = async function (deployer, account, assetid) {
+  // note: if you have an indexer instance available it is easier to just use this
+  //     let accountInfo = await indexerClient.searchAccounts()
+  //    .assetID(assetIndex).do();
+  // and in the loop below use this to extract the asset for a particular account
+  // accountInfo['accounts'][idx][account]);
+  let accountInfo = await deployer.algodClient.accountInformation(account).do();
+  for (idx = 0; idx < accountInfo['created-assets'].length; idx++) {
+    let scrutinizedAsset = accountInfo['created-assets'][idx];
+    if (scrutinizedAsset['index'] == assetid) {
+      console.log("Created ASA:", {
+        assetIndex: assetid,
+        params: scrutinizedAsset['params']
+      })
+      break;
+    }
+  }
+};
+
+// Print specific account's ASA info.
+exports.printAssetHolding = async function (deployer, account, assetid) {
+  // note: if you have an indexer instance available it is easier to just use this
+  //     let accountInfo = await indexerClient.searchAccounts()
+  //    .assetID(assetIndex).do();
+  // and in the loop below use this to extract the asset for a particular account
+  // accountInfo['accounts'][idx][account]);
+  let accountInfo = await deployer.algodClient.accountInformation(account).do();
+  for (idx = 0; idx < accountInfo['assets'].length; idx++) {
+    let scrutinizedAsset = accountInfo['assets'][idx];
+    if (scrutinizedAsset['asset-id'] == assetid) {
+      console.log("Asset Holding Info:", scrutinizedAsset, accountInfo);
+      break;
+    }
+  }
+};
+
+// Print account's ASA info. All at once.
+exports.printAssets = async function (deployer, account) {
```

Comment:
as above, let's use  `algodClient` directly. same for other functions below.

Reply:
In some functions I used `Deployer.waitForConfirmation` so it's better not to mix

Reply:
OK